ui/app/src/components/LauncherSiteCard/LauncherSiteCard.tsx

```diff
@@ -75,22 +79,29 @@ function LauncherSiteCard(props: LauncherSiteCardProps) {
     <>
       <Box
         // @ts-ignore
-        button
+        button={isSiteReady}
```

Comment:
Migrate/use ListItemButton. Ts ignore should be removable then; otherwise, add comment on what's the ts-ignore for.